stock_exchange/apps/stock/models.py

```diff
@@ -1,4 +1,77 @@
 from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from datetime import date
```

Comment:
This import should go up. By this i mean:

```
from datetime import date

from django.db import models
from django.utils.translation import ugettext_lazy as _
```

Why? Because python base modules always go before 3 party libraries and datetime module is a standard python module.